README.md

```diff
@@ -83,26 +83,36 @@ By default, you should start composing new scripts and test using forked network
 You have three forked networks to work with:
 
 - `mainnet-fork`
-- `goerli-fork`
 - `holesky-fork`
+- `sepolia-fork`
 
 To start new voting on the live networks you could proceed with:
 
 - `mainnet`
-- `goerli`
 - `holesky`
+- `sepolia`
 
-> Note: you can't run tests on the live networks.
+> [!CAUTION]
+> You can't run tests on the live networks.
 
 In a typical weekly omnibus workflow, you need only `mainnet-fork` and
 `mainnet` networks. In case of large test campaign on Lido upgrades,
-it also could be useful to go with `goerli` and `goerli-fork` testnets first.
+it also could be useful to go with `holesky` and `holesky-fork` testnets first.
 
+> [!IMPORTANT]
 > **Holesky specifics.**
 > Due to Holesky not being supported by Infura yet, setting RPC URL for Holesky is different. Instead of setting `WEB3_INFURA_PROJECT_ID` env variable set `HOLESKY_RPC_URL`.
```

Comment:
https://docs.infura.io/api/network-endpoints#ethereum

At the moment of writing, Infura supports both Holešky and Sepolia :eyes:

Reply:
Great! Updated network config. Re-mapped to Infura RPC.